bindinfo/bind_test.go

```diff
@@ -1853,8 +1853,8 @@ func (s *testSuite) TestIssue19836(c *C) {
 		PS: []*util.ProcessInfo{tk.Se.ShowProcess()},
 	})
 	explainResult := testkit.Rows(
-		"Limit_8 2.00 0 root  time:0s, loops:0 offset:1, count:2 N/A N/A",
-		"└─TableReader_14 3.00 0 root  time:0s, loops:0 data:Limit_13 N/A N/A",
+		"Limit_8 2.00 0 root   offset:1, count:2 N/A N/A",
```

Comment:
Explain for connection, the RuntimeStatsColl is overwritten, so the `time:0s, loops:0` is gone.

```
select ... from ...
explain for connection ... // ResetContextOfStmt overwrite the previous statement context, so there is no runtime stats coll
```

Reply:
That does not explain why it is changed. `ResetContextOfStmt` should overwrite the previous statement context, before this PR.